src/App.vue

```diff
@@ -69,6 +69,7 @@
     <div class="app-body">
       <!-- only show pages while signing in or once the data is loaded -->
       <main v-if="isSigninRoute || dataLoaded">
+        <BreadCrumb :breadcrumbs="breadcrumbs" />
```

Comment:
You know how words sometimes look funny? "crumb" is doing it for me atm 🤣

Reply:
haha i know, it's such a weird word. I'm not sure if it's meant to be one or two words..